### DISCUSSION TITLE
从数字经济学视角构建分析开源的框架

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#408] [1️⃣ 新鲜事]
前面思考过从法律（特别是[知识产权](https://github.com/orgs/X-lab2017/discussions/406)）的视角看开源发展的脉络。一个更加热门和宏大的视角即是当下的**数字经济学**。通过市面上流行的几本数字经济学读本中的框架，看看怎么和开源发展的分析进行结合。

《[**数字经济概论**](https://item.jd.com/13021905.html)》是一本 2022 年出版，相对较早的一本，可以参考的要点包括：
- **历史脉络**：信息经济（1962）--> 数字化生存（尼葛洛庞帝，1995）--> 数字经济（Tapscott，1996）
- **数字经济的“三化”**：
  - 数字产业化：互联网、数据驱动、网络化协同
  - 产业数字化：平台 + 生态、线上 + 线下、分享型经济、纵向一体化
  - 数字化治理：民主参与、集体协作、去中心、自组织、治理体系数字化、产业结构数字化
- **数字经济的理论分析框架**
  - 生产过程：提升经济效率（精细分工与协同）
  - 交易过程：降低运行成本（交易成本）
  - 扩散过程：推动组织创新（溢出效应、正反馈、路径依赖）
  - 基础支撑：产权基础、产权组合、产权激励、产权表现
- **数字经济的生态体系**
  - 创新体系：技术、模式、要素、主体
  - 产业体系：数字产业、产业融合、数字基建
  - 治理体系：政策、法律、监管、安全
- **数字经济成为全球普遍共识**
  - 加强数字经济战略布局
  - 多边机制成为数字经济的重要议题
  - 强化数字技术创新能力
  - 深化数字技术融合创新应用
  - 提升政府数字治理能力
  - 保障数字经济安全
  - 注重消弭数字鸿沟
- **数字经济创新理论框架**
  - 技术创新
  - 服务创新
  - 组织创新
  - 治理创新
- **数字贸易化**
  - 产品内容云端化
  - 数字产品可交易化
  - 产品内容高度复用，边际成本为零
  - 产品内容交互化
- **产业数字化转型的框架**
  - 构成逻辑：技术转型 --> 业务转型 --> 制度革新
  - 技术转型：基础，提供赋能工具，拓展业务的广度与深度
  - 业务转型：核心，革新产业链，提高生产效率、减低成本、控制风险
  - 制度转型：保障
- **产业数字化转型空间度量模型**
  - 交易费用：通过数字技术降低
  - 产业分工与生产协同：通过数字技术提升效率
  - 网络外部性：通过数字技术进一步加强
  - 产权变革：更加优化的资源配置

《[**数字经济：内涵与路径**](https://item.jd.com/13335165.html)》是一本非常不错的读本：
- 分析视角：技术、市场、治理
  - 将三个视角统一，围绕如何激活数据要素，发展数字生产力，建立数字化生产关系，这一数字经济发展主线
  - 数字经济是技术经济学，也是政治经济学，需要从生产力和生产关系的视角来深刻理解
  - 面向数字经济热点问题，有针对性地构建了**平台经济**、**产业互联网**、**产业数字金融**和**数字治理**的知识架构
  - **数字经济**：是指人类在**全球化数据网络**基础上，利用各种**数字技术**，通过数据处理来优化**社会资源配置**、创造**数据产品**、形成**数据消费**，进而创造人类的**数据财富**、推动**全球生产力**发展的经济形态。
  - 三大特点：平台支撑、数据驱动、普惠共享
  - 内在逻辑：一种广泛连接的社会经济系统，**二维市场**（实体、数字） + **连接资源**（利益相关者、产品、服务等），构成了数字经济存在的价值基础
- 框架构成：
  - 数字生产力
    - 生产力基本要素：劳动资料（生产工具）、劳动对象、劳动者、科学技术
    - 数字生产力**定义**：指在数字经济时代，人类在创造财富的过程中所用到的**数字化工具**（硬件、软件、算法等）、**数字对象**（数据、连接、信用等）和**数字生产者**（分析师、开发者、设计师等）。
    - 数字生产力**构成要素**：**算法**（劳动资料）、**连接**（劳动对象）、**分析师**（劳动者）
    - 数字生产力的**五全特征**：全空域、全流程、全场景、全解析、全价值
    - 数字生产力发展的新基建：**信息基础设施**（软硬件）、**融合基础设施**（传统产业数字化）、**创新基础设施**（创新体系）
  - 数字生产关系
    - **生产关系**：是人们在物质生产过程中结成的经济关系，还有的其它社会关系包括政治关系、家庭关系、宗教关系等。
      - 从静态看，有生产资料所有制关系、生产中人与人的关系和产品分配关系构成
      - 从动态看，有生产、分配、交换、消费四个环节构成
      - 基本要素：生产资料所有制、生产中的地位与交换关系、产品分配方式
      - 生产关系所包含的社会关系：劳动价值关系、经济权力关系、经济利益关系
    - 数字化工作方式：组织在线、沟通在线、协同在线、业务在线、生态在线
    - 数字化生产关系特征：透明性、可信性、对等性
  - 数据要素
    - 市场秩序理论：要素只有在统一开放、竞争有序的市场上配置，市场调节资源配置才是有效的，成本才是最低的。
    - 要素市场的功能：规范要素市场秩序、发现要素市场价格、配置要素市场结构
    - 数据成为要素的意义：数据参与生产、数据参与分配、数据融合带动
    - 从数据到数据要素（数据要素化）：数据资源化、数据资产化、数据资本化
    - 数据要素的定义：指进行社会生产经营活动时所需要的各种社会资源，是维系国民经济运行及市场主体生产经营过程中所必须具备的基本因素。
    - 数据要素的特征：非竞争性、非排他性、非耗竭性、非稀缺性、非恒价性
    - 要素数据化：一方面是传统要素（土地、资本、科技、劳动力）本身的数字化，另一方面，传统要素在数字空间里会产生“新土地”“、”新劳动力“、”新资本“、”新技术“，从而丰富传统要素的内容和市场化方式。
    - 科技要素与数据要素融合：进一步提高科技研发的效率与效果，包括
      - 基于数据要素的**科技创新基础设施**：算力网
      - 基于数据要素的**科技创新模式创新**：开放创新平台（创新链、产业链、价值链） + 开源生态
      - 基于数据要素的**科技成果转化新模式**：原始基础创新、技术转化创新、大规模生产
    - 数字化劳动力：
      - 提高劳动力数字化能力素养
      - 优化劳动力数字化发展环境
      - 探索培养数字化新劳动力
      - 加快培育数字经济专业人才
  - 数字化应用（互联网平台、金融）
    - **产业互联网**：通过产业内各个参与者的互联互通，改变产业内在数据采集、流通和使用的方式，通过为每一个环节提供可信的数字赋能，改变每个环节创造价值的方式，并最终改变产业生态的运转方式。
    - **数字金融**：利用金融科技构建的金融新生态。
  - 数字化治理
    - 数字治理：政府采取数字化方式，推进数据信息共享和政务数字化公开，并在此基础上，通过数字治理解决社会发展的治理命题，即利用数字化手段更加全民地考察政府行政行为产生的效果，采取有效的数据分析方法提高政府政府对政策和措施效果的精准评估能力，尽可能地辅助政府做出符合公共利益的价值判断。
    - 通过数字化、智能化赋能，提升社会治理的科学性、透明性、民主性、多元性和包容性，进而提升社会治理的效能。
  - 最后一点（2022年4月，习书记）：加快构建中国特色哲学社会科学，归根结底是构建中国自主的知识体系。努力把中国优秀传统文化思想与数字技术相结合，推动数字经济领域的**知识创新**、**理论创新**、**方法创新**，从而形成具有中国特色的数字经济理论。

《[**数字经济的逻辑**](https://item.jd.com/13400637.html)》也较为系统的介绍了数字经济的框架：
- TODO

《[**数字经济学**](https://item.jd.com/14165772.html)》则是一本由高校教师编撰面向大学生的读物，很多东西都涉及一点的大杂烩，比较适合入门读物，深入一点的还需要结合其他内容看，对于不熟悉该领域的有一定启发性。以下列出目录和要点，供参考。

**第一部分 新动能：数字经济的缘起**
- **第一章 数字经济的起源与发展逻辑**
  - 第一节 数字经济概念的梳理
    - 定义：狭义（数字产业）、广义（数字经济活动）
    - 四化框架：数字产业化、产业数字化、数字化治理、数据价值化
    - 组成：（数据 + 数字技术）-->（数字产业化）-->（产业数字化 + 数字化治理）--> （数据价值）
    - 数字经济与传统经济对比
      - 传统经济：L（劳动力）+ K（资本）
      - 数字经济：L（劳动力）+ K（资本）+ D（数据）
  - 第二节 数字经济的发展历程
  - 第三节 数字经济发展的意义
  - 第四节 数字经济分析框架
    - 分析框架（七个新）：新动能、新基建、新组织、新模式、新要素、新冲击、新格局

**第二部分 新基建：数字基础设施**
- **第二章 数字基础设施与数字赋能**
  - 第一节 从传统信息技术到数字技术
  - 第二节 新型数字基础设施
  - 第三节 数字基础设施的发展与未来

**第三部分 新组织一：双边平台和消费互联网**
- **第三章 平台经济和双边市场介绍**
  - 第一节 平台的结构
    - 平台经济的结构：平台企业 + 在线市场
  - 第二节 平台的分类
  - 第三节 双边平台的发展逻辑
- **第四章 双边平台内的经济现象**
  - 第一节 网络效应与用户效用
  - 第二节 产品定价与价格结构
  - 第三节 黏性与锁定
  - 第四节 搜寻与匹配
  - 第五节 平台分层
  - 第六节 信息披露
  - 第七节 从双边到多边
  - 第八节 经济学建模：网络效应的模型化
- **第五章 双边平台的竞争与合作**
  - 第一节 平台竞争
  - 第二节 多归属问题
  - 第三节 流量竞争
  - 第四节 平台兼容
- **第六章 双边平台的生态化与国际化**
  - 第一节 生态互补者
    - 平台生态（Platform Ecosystem）：以平台企业为核心企业（Facal Firm），由联结在数字平台上的企业、用户、政府、产业联盟等各类具有不同利益诉求的互补者（Complementors）所构成的复杂商业生态系统。
  - 第二节 平台生态治理
    - 治理主体：平台企业
    - 治理参与者：平台生态内部的利益互补者
    - 治理对象：平台生态的各类经济社会价值
    - 核心逻辑：实现平台生态的长期利益最大化和持续、创新、健康发展
  - 第三节 平台跨国竞争
- **第七章 平台经济反垄断**
  - 第一节 平台经济反垄断的来龙去脉
    - 垄断类型：生态垄断、技术垄断、数据垄断
  - 第二节 双边平台垄断行为的经济学分析
  - 第三节 平台经济反垄断的影响
- **第八章 消费互联网**
  - 第一节 消费互联网简介
    - 定义：为满足用户日益增长的各类生活消费的需求，及部分企业更加便捷地出售各类产品和服务的需求，依托用户消费数据，并利用5G、匹配、定位、大数据等数字技术，所形成的链接消费市场中各类用户、企业的数字化消费网络。
  - 第二节 消费互联网的运营模式
    - 模式：流量驱动型、电商交易型、本地生活型
  - 第三节 消费互联网：共享经济的热潮
    - 经济模式的分类：传统经济、租赁经济、共享经济

**第四部分 新组织二：产业平台和产业互联网**
- **第九章 产业平台初探**
  - 第一节 产业平台简介
    - 产业平台（Industry Platforms）：源自各类数字技术，通过数字技术开发出供不同类型企业使用的共性功能，从而让这些企业基于对共性功能的需求联接在平台上，平台上的企业是平台的互补者。
    - 分类：技术授权型（如芯片）、开源社区型（如鸿蒙）、技术联盟型（如5G）
    - 产业平台和双边平台的区别
      - 产业平台：共同的技术构件，如 Windows、Android、Pytorch、Intel、GitHub、腾讯云等
      - 双边平台：中介或在线市场，如优步、苹果商店、抖音、美团、爱彼迎等
      - 混合平台：兼具上面两者，如苹果、谷歌、微软、脸书、腾讯、亚马逊等
  - 第二节 典型的产业平台案例
  - 第三节 产业平台的经济学现象：开发者与产业平台
- **第十章 产业互联网**
  - 第一节 产业互联网的定义
    - 定义：产业互联网（Industrial Internet）：是一个具有打通全产业链、联通供求端数据和其他要素，以实现产业数字化的网络。
      - 狭义（工业互联网）：最开始提出的，仅仅局限于工业行业内的工业互联网，仅涉及工业生成活动。
      - 广义（产业互联网）：指拓展至包括非工业行业在内的互联网，涉及工业、服务业、农业等生产活动。
    - 相较于工业互联网，产业互联网这个说法更准确，因为其开始跳出工业生成本身，拓展到各行各业的联接中
    - 系统的给出相关概念的定义：
      - **消费互联网**：为满足用户日益增长的各类生活消费需求，以及部分企业更加便捷地出售各类产品和服务的需求，依托用户消费数据，并利用 5G、匹配、定位、大数据、人工智能等数字技术，所形成的连接消费市场中各类用户、企业的数字化消费网络。
      - **工业互联网**：为满足工业企业提升生产效率、实现运营协同的需求，依托工业经验知识（know-how）和生成数据，并利用智能机器、工业软件、云计算、人工智能、5G等数字技术，所形成的联结工业企业内部生产单元乃至工业产业链中各类其他企业的数字化生产网络。
      - **产业互联网**：为满足各行业企业提升生产效率、按需调整生产活动、实现运营协同的需求，依托行业经验知识和生成数据，并利用智能机器、行业软件、云计算、人工智能、5G 等数字技术，所形成的联结行业企业内部生产单元和企业外部产业链合作伙伴的数字化生产网络；也是国家促进传统产业升级，打造数字创新生态，实现数字化转型的新型产业模式。
  - 第二节 产业互联网的框架
  - 第三节 产业互联网的发展类型

**第五部分 新模式：数字生态**
- **第十一章 数字生态的简介**
  - 第一节 数字生态的源起
    - 新经济平台不断崛起：供应链模式 --> 平台模式 --> 生态模式
    - STEP 模型：用户结构（Structure）、交易层级（Transaction tier）、商业赋能（Enablement）、模式绩效（Performance）
  - 第二节 数字生态的定义
    - 数字生态定义：由数字产业中交互的各类组织、企业和个人共同支撑的一个数字产业共同体。其成员囊括了一系列的利益相关者，包括政府、行业协会、供应商、生产商、竞争对手、客户等，这些利益相关者在整个生态中共同演化，分享愿景，发展解决方案，相互信任，从而形成数字共同体。
    - 三大流派：商业生态系统、创新生态系统、平台生态系统
  - 第三节 数字生态的结构
    - 组成：产业价值网络 + 泛社区网络
    - 差序格局：产业价值网络（核心企业 --> 领袖伙伴 --> 其他伙伴）-->  泛社区网络（政府、机构、高校、联盟、中介等）
    - 产业价值网络的价值链组成：数字基础设施、数据要素市场、产业互联网、消费互联网、数字收益分配
  - 第四节 数字生态的培育
    - 6C 框架：运营情景（Context）、结构（Construct）、运行机制（Cooperation）、范式（Configuration）、能力（Capability）、变革（Change）
  - 第五节 数字生态的商业模式
    - 定义：在一个数字生态系统中，不同利益相关者可协同、可演化的多个商业模式共同集成的商业模式，是商业模式的商业模式。
    - 4 Co 框架：价值社区（Value community）、价值共设（Value co-design）、价值共创（Vaule co-creation）、价值获取（Vaule co-capture）

- **第十二章 数字生态的竞争**
  - 第一节 信息产业与数字产业
  - 第二节 中美数字生态的竞争
  - 第三节 数字生态的未来博弈策略

**第六部分 新要素：数据要素市场**
- **第十三章 数据生态**
  - 第一节 数据的定义和特征
  - 第二节 数据生态的解构
    - 定义：围绕数据产业发生交互的各类组织、企业和个人共同支撑的一个数据产业共同体。
    - 数据产业价值网络：授权、采集、归集、存储、加工、分析、消费
  - 第三节 数据算法
    - 数据挖掘：侧重描述数据的应用
    - 机器学习：强调在数据中发现新知识或新模式所采用的方法
    - 隐私计算：关注数据加工建模过程中的隐私保护问题
- **第十四章 数据的确权**
  - 第一节 数据确权的现实背景
  - 第二节 数据确权的理论背景
  - 第三节 数据分级授权带来的福利
  - 第四节 影响数据分级授权效果的因素
  - 第五节 数据分类分级授权的制度设计
- **第十五章 数据的定价与交易**
  - 第一节 数据定价
    - 借鉴会计学的资产定价方法：市场法、成本法、收入法
    - 数据定价的机制：固定定价、差别定价、自动实时定价、协商定价、拍卖定价
  - 第二节 数据要素市场交易体系
  - 第三节 分类分级标识体系构建思路
- **第十六章 数据的治理与监管**
  - 第一节 国内外数据治理和跨境流通现状
  - 第二节 数据要素市场面临的问题
  - 第三节 数据要素市场分类分级的监管体系

**第七部分 新冲击：数字经济的宏观影响**
- **第十七章 数字经济和经济发展**
  - 第一节 数字经济对宏观经济活动的影响
  - 第二节 数字经济下宏观经济活动的特点
- **第十八章 数据要素和经济增长**
  - 第一节 生产要素与经济增长：一个历史的视角
  - 第二节 数据要素如何影响经济增长
  - 第三节 数据要素和经济增长：宏观经济模型简介
- **第十九章 数字经济的宏观贡献**
  - 第一节 国外数字经济规模的测算方法及结果
  - 第二节 国内数字经济规模的测算方法及结果
  - 第三节 数字经济测算的其他视角
- **第二十章 人工智能与就业**
  - 第一节 人工智能时代
  - 第二节 人工智能的国际格局
  - 第三节 人工智能、机器人与就业
- **第二十一章 大数据与绿色发展**
  - 第一节 大数据时代
  - 第二节 数字经济与“双碳”目标
    - 开源（促进） --> 数字经济（促进）--> 绿色发展
  - 第三节 大数据赋能绿色发展：以公共数据为例
- **第二十二章 数字货币与数字税费**
  - 第一节 数字货币简介
    - 种类：匿名币、稳定币、央行数字货币
  - 第二节 数字货币的应用场景
  - 第三节 数字税费的设计理念

**第八部分 新格局：数字经济的区域扩张**
- **第二十三章 数字经济与区域经济发展**
  - 第一节 数字经济赋能区域经济发展的机制
  - 第二节 数字城市的演绎路径
  - 第三节 数字鸿沟扩大了还是缩小了？
- **第二十四章 数字经济全球化**
  - 第一节 从工业全球化到数字全球化
  - 第二节 新动能：数字贸易
  - 第三节 新组织：平台型跨国企业
  - 第四节 新模式：数字生态国际化
  - 第五节 数字全球化的未来展望

**Comment by Wuhanqin** [comment]
最近北京科技大学陈晓红老师出了一本新书，讲开源经济的，做了一些偏经管数学建模的实证研究，可以结合看看。模型还是很不错的，传统and经典。
